src/components/UserProfile.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div class="w-full bg-amber-400 p-8">
+    <div class="bg-white rounded p-16 flex justify-between gap-44" v-show="!showQR">
+      <div>
+        <h3 class="font-semibold text-2xl capitalize">personal page</h3>
+        <ul class="list-none mt-4 space-y-2">
+          <li class="text-wrap break-words">ID: {{ userInfo?.id }}</li>
+          <li>Name: {{ userInfo?.name }}</li>
+          <li>Email: {{ userInfo?.email }}</li>
+        </ul>
+      </div>
+      <div>
+        <h3 class="font-semibold text-2xl capitalize">mobile app authenticaiton (2FA)</h3>
+        <p class="mt-2">Secure your account with TOTP two factor authentication</p>
+        <div class="mt-2">
+          <button
```

Comment:
To create a uniform look across the app, let's extract this button to a button component. This button component should be able to handle whatever I pass within the tags

Hint: Slots